config/services.yaml

```diff
@@ -50,6 +50,7 @@ services:
         $cacheEnabled: true
 
     Ilios\MeSH\Parser:
+      public: true
```

Comment:
We haven't done it in a while, but I think the place to do this when it's only needed for testing is in `config/services_test.yaml`.

Reply:
nice, that's much better. thanks.

Reply:
:heavy_check_mark:  handled.